quickbudget/api/serializers.py

```diff
@@ -0,0 +1,90 @@
+from rest_framework import serializers
+
+from quickbudget import models
+from quickbudget.models import Budget, Expense, Category, User
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ["id", "username", "password"]
```

Comment:
you need to create another serializer for responses. you don't need to return the password (even if it's hashed).

Reply:
also, look at the `.to_representation` method for the BaseSerializer class.